internal/vfs/vfstest/vfstest.go

```diff
@@ -17,7 +18,7 @@ import (
 	"github.com/microsoft/typescript-go/internal/vfs/iovfs"
 )
 
-type mapFS struct {
+type MapFS struct {
```

Comment:
If `MapFS` is exported, you could probably just make `FromMap` and `FromMapWithClock` export `*MapFS` directly. Optional, though.

Reply:
It cant be though right. it returns `ioFS` which has MapFS

Reply:
unfortunately the way its wrapped it becomes like that.